docs/wiki/01 - STUDY PARAMETERS SETUP/01 - Study Variables/04 - Subprojects.md

```diff
@@ -0,0 +1,45 @@
+# Subprojects
+
+## Overview
+The **Subproject** plays an important role in determining the instrument battery that will be assigned to a candidate at a timepoint. **Subprojects** are defined in the `subproject` table of the database.
+
+## Adding Subproject Options
+
+### Front End (Recommended)
+Subprojects are defined in the Configuration module, which can be found in LORIS under the **Admin** menu tab.  Click on _To configure study subprojects click here_ link at the top of the page. Refer to the help section of the module for further instructions on how to add or modify subprojects.
+   
+### SQL
+Subprojects can be added directly in SQL using the following command.
+
+```sql
+INSERT INTO subproject (title) VALUES('SCI');
+```
+
+
+### API
+ _not yet available. See [API documentation](../../../API/) for latest additions_
```

Comment:
This should be a hard link to the master branch, not the current branch, if you really want people to look at the latest merges in API docs.

Reply:
i dont think so this documentation will only be released with the documentation of the API, they are on the same branch.

Reply:
ok, well then remove `for latest additions`

Reply:
i dont see why, the changes on that branch would be the latest changes released..

Reply:
it's redundant, but up to you.